nsv13/code/modules/munitions/munitions.dm

```diff
@@ -361,9 +376,22 @@
 	chambered = null
 	loaded = null
 	firing = FALSE
+	maint_req --
 	if(proj_type) //Looks like we were able to fire a projectile, let's tell the ship what kind of bullet to shoot.
 		return proj_type
 
+/obj/structure/ship_weapon/proc/weapon_malfunction()
+	malfunction = TRUE
+	playsound(src, 'sound/effects/alert.ogg', 100, 1) //replace this with appropriate sound
```

Comment:
fourth arg should be TRUE iirc its a bool

Reply:
TBH it's done a lot elsewhere in code, but yes, it is a bool. Defaults to TRUE anyway.

Reply:
thats because back in 2017 we didnt use bool defines and cyberboss fixed the biggest offendors and made it a contribution guideline, but playsound wasnt taken in account, in general this is now expected in PRs

Reply:
resolved: changes incorporated